plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -186,10 +192,14 @@ export class PersonState {
         teamId: number,
         isUserId: number | null,
         isIdentified: boolean,
-        uuid: string,
         creatorEventUuid: string,
-        distinctIds?: string[]
+        distinctIds: string[]
     ): Promise<Person> {
+        if (distinctIds.length < 1) {
+            throw new Error('at least 1 distinctId is required in `createPerson`')
+        }
+        const uuid = uuidFromDistinctId(teamId, distinctIds[0])
```

Comment:
Can we really have multiple distinct_ids during creation time? If yes we need to add overrides for these or is that covered somewhere else?

Reply:
This isn't made possible by this PR, but yes in prod today you can merge 2 `distinct_id`s that don't even exist in the DB yet, they will of course then point to one `person_id`.

This PR doesn't cover not creating PG/CH rows, but there may need to be provisions for this case in that PR.